charts/bulk-scan-orchestrator/values.yaml

```diff
@@ -75,6 +76,8 @@ java:
     FLYWAY_SKIP_MIGRATIONS: "true"
     ENVELOPE_QUEUE_CHECK_POLL_INTERVAL: "PT1H"
     CDAM_ENABLED: "true"
+    PAYMENT_PROCESSING_INTERVAL: "10000"
```

Comment:
Would this be good to have as an environment variable? In the case that there is logging within the methods, we may want to increase this for lower environments.

Reply:
added as a environment variable